ajlineb/pull-request-demo
# Modified README

To test:

- Open the index.html in the browser
- Click the button
- Verify that the alert opens with the message "Welcome to my app"
